src/TropicalGeometry/TropicalPolyhedron/properties.jl

```diff
@@ -0,0 +1,366 @@
+that is the zero-dimensional cells in the covector decomposition of `P`.
+
+# Examples
+The following retrievs the pseudovertices of a bounded tropical polytope:
+```jldoctest
+julia> P = tropical_convex_hull(QQ[0 -1 0; 0 -4 -1]);
+
+julia> vertices(P)
+2-element SubObjectIterator{PointVector{TropicalSemiringElem{typeof(min)}}}:
+ [(0), (-1), (0)]
+ [(0), (-4), (-1)]
+
+julia> pseudovertices(P)
+3-element SubObjectIterator{PointVector{TropicalSemiringElem{typeof(min)}}}:
+ [(0), (-1), (0)]
+ [(0), (-4), (-1)]
+ [(0), (-3), (0)]
+```
+
+For an unbounded tropical polytope, this only outputs the points for which all coordinates are finite:
+```jldoctest
+julia> TT = tropical_semiring();
+
+julia> P = tropical_convex_hull(TT[0 1 4; inf 0 2; inf inf 0]);
+
+julia> pseudovertices(P)
+2-element SubObjectIterator{PointVector{TropicalSemiringElem{typeof(min)}}}:
+ [(0), (1), (4)]
+ [(0), (1), (3)]
+```
+"""
+function pseudovertices(as::Type{PointVector{T}}, P::TropicalPolyhedron) where {T<:TropicalSemiringElem}
+  CT = pm_object(P).PSEUDOVERTEX_COARSE_COVECTORS::Polymake.Matrix{<:Integer}
+  n = count(!iszero, eachrow(CT))
+
+  return SubObjectIterator{as}(P, _pseudovertices, n)
+end
+
+function pseudovertices(as::Type{PointVector{T}}, P::TropicalPointConfiguration) where {T<:TropicalSemiringElem}
+  n = n_pseudovertices(P)
+
+  return SubObjectIterator{as}(P, _pseudovertices, n)
+end
+pseudovertices(P::Union{TropicalPolyhedron{M},TropicalPointConfiguration{M}}) where {M<:MinOrMax} = pseudovertices(PointVector{TropicalSemiringElem{M}}, P)
+
+n_pseudovertices(P::TropicalPointConfiguration) = pm_object(P).PSEUDOVERTICES |> nrows
+function n_pseudovertices(P::TropicalPolyhedron) 
+  CT = pm_object(P).PSEUDOVERTEX_COARSE_COVECTORS::Polymake.Matrix{<:Integer}
+
+  return count(!iszero, eachrow(CT))
+end
+
+function _pseudovertices(::Type{PointVector{TropicalSemiringElem{M}}}, P::Union{TropicalPolyhedron,TropicalPointConfiguration}, i::Int) where {M<:MinOrMax}
+  T = tropical_semiring(convention(P))
+
+  return point_vector(
+    T,
+    T.(pm_object(P).PSEUDOVERTICES[i,:])
+  )
+end
+
+@doc raw"""
+    points([as::Type{T} = PointVector,] P::TropicalPointConfiguration)
+
+Return an iterator over the points of tropical point configuration `P`.
+
+# Examples
+The following retrievs the pseudovertices of a bounded tropical polytope:
+```jldoctest
+julia> P = tropical_point_configuration(QQ[0 -1 0; 0 -4 -1]);
+
+julia> points(P)
+2-element SubObjectIterator{PointVector{TropicalSemiringElem{typeof(min)}}}:
+ [(0), (-1), (0)]
+ [(0), (-4), (-1)]
+
+```
+"""
+points(P::TropicalPointConfiguration{M}) where {M<:MinOrMax} = points(PointVector{TropicalSemiringElem{M}}, P)
+function points(as::Type{PointVector{T}}, P::TropicalPointConfiguration) where {T<:TropicalSemiringElem}
+  n = n_points(P)
+
+  return SubObjectIterator{as}(P, _points, n)
+end
+
+@doc raw"""
+    n_vertices(P::TropicalPointConfiguration)
```

Comment:
```suggestion
    n_points(P::TropicalPointConfiguration)
```
(according to the doctest and code below)